packages/ppx/test/snapshot/reason/melange-styles-prop-expansion.t/dune

```diff
@@ -0,0 +1,7 @@
+(melange.emit
+ (alias melange-styles-prop-expansion)
```

Comment:
why alias in a cram test?

Reply:
I think there is no reason. I just copied and pasted from this cram test.
https://github.com/davesnx/styled-ppx/blob/8703b481b4d0872aaeb330cb20d5e3bf607d0430/packages/ppx/test/snapshot/reason/melange-emit-only.t/dune#L19-L25
Going to remove it.